src/app.js

```diff
@@ -102,25 +103,65 @@ go.app = function() {
                             section_name : choice.label,
                             section_id : choice.value,
                             opts_data : opts.data,
-                            location_name : opts.full_name
+                            location_name : opts.full_name,
+                            location_id : opts.full_geoid
                         }
                     };
                 }
             });
         });
 
+        function sub_section(data, section_id) {
+            return sub_section[section_id](data);
+        }
+
+        sub_section.elections = function(data) {
+            //I would like to loop through the party distribution to return the top 3 parties
```

Comment:
Is the difficulty sorting them? If so you can use lodash's `orderBy` (https://lodash.com/docs#orderBy) or `sortBy` (https://lodash.com/docs#sortBy) methods.